pkg/nodeconfig/nodeconfig.go

```diff
@@ -564,12 +564,7 @@ func generateKubeletConfiguration(clusterDNS string) kubeletconfig.KubeletConfig
 		KubeAPIBurst:          100,
 		SerializeImagePulls:   &falseBool,
```

Comment:
Don't you need to add `EnableSystemLogQuery` here?

Reply:
`oc adm node-logs` definitely cannot work without that being set. It is being set in the [MCO kubelet conf](https://github.com/openshift/machine-config-operator/pull/3645). Is that getting pulled in by WMCO?